### PR TITLE
Improved descriptions and instructions

### PR DESCRIPTION
**Pull request by Indy9000**
Improved descriptions and instructions

### CODE REVIEW

**Comment by Indy9000** [comment]
Why is this failing? Only changed the .md

**Comment by forki** [comment]
Don't worry. The build is not supposed to work since it's pre-init stage.

**Comment by Indy9000** [comment]
ah ok :) cool

File: docs/content/index.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
-# ProjectScaffold
+##What is ProjectScaffold?
+ProjectScaffold helps you get started with a new F# .NET/Mono project solution with everything needed for successful organising of code, tools and publishing.
```
Comment (forki) [inline docs/content/index.md]: Please remove the emphasis on F#.  We want to make this a C#/VB/F# project.

File: docs/content/index.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+ProjectScaffold helps you get started with a new F# .NET/Mono project solution with everything needed for successful organising of code, tools and publishing. 
 
-## Getting started
+We recommend you use ProjectScaffold for all new projects.
 
-Used to initialialize a prototypical .NET/mono solution including file system layout and tooling.
+Tools include [Paket](paket-package-management.html) for dependency management and [FAKE](fake-build.html) for automating the build process 
 
-This includes [Paket](paket-package-management.html) dependency management and a [FAKE](fake-build.html) build process that: 
+Packet -
```
Comment (forki) [inline docs/content/index.md]: It's "Paket"

File: docs/content/index.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
 
-## Getting started
+We recommend you use ProjectScaffold for all new projects.
 
-Used to initialialize a prototypical .NET/mono solution including file system layout and tooling.
+Tools include [Paket](paket-package-management.html) for dependency management and [FAKE](fake-build.html) for automating the build process 
 
-This includes [Paket](paket-package-management.html) dependency management and a [FAKE](fake-build.html) build process that: 
+Packet - 
+* helps manage Nuget packages
```
Comment (forki) [inline docs/content/index.md]: And "NuGet"  sorry for being pedantic

**Comment by forki** [comment]
Very cool. Only small spelling suggestions

File: docs/content/index.md (hunk 1 above)
Comment (Indy9000) [inline docs/content/index.md]: Sure, but wouldn't it be clearer if we have F# now and other langs added when they become available.

Comment (forki) [inline docs/content/index.md]: They are already working. It's just not created during init yet.

**Comment by forki** [comment]
thanks a lot

**Pull request merged**